src/main/kotlin/plugins/scijava/MastodonSidePlugin.kt

```diff
@@ -78,8 +81,7 @@ class MastodonSidePlugin : DynamicCommand() {
         var useThisResolutionDownscale = "[1,1,1]"
         lateinit var levelNames: MutableList<String>
         fun levelParams() {
-            val chSource = mastodon!!
-                .sharedBdvData.sources[channelIdx].spimSource
+            val chSource = mastodon.sharedBdvData.sources[channelIdx].spimSource
```

Comment:
silently assuming `sources.getSize() > 0`... but that's an issue from early days, not related to this PR.. just caught my attention